.changeset/little-cars-impress.md

```diff
@@ -0,0 +1,8 @@
+---
+"@logto/console": minor
+"@logto/phrases": minor
+"@logto/core": minor
+"@logto/ui": minor
+---
+
+New feature: User account settings page
```

Comment:
1. `.changeset-staged`?
2. should we write a detailed changelog here?

Reply:
Ah, my bad, sorry.
And I'm not sure if we should only mention the non-cloud features in this changelog. Let's talk about it on Slack when you have time

Reply:
Discussed offline and the changeset file has been updated.